kubelet/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG - kubelet
 
+## 4.0.0 / 2020-01-13
+
+* [Fixed] Improve url join to not mutate the base url when proxying a call. See [#5416](https://github.com/DataDog/integrations-core/pull/5416).
+* [Added] Make OpenMetrics use the RequestsWrapper. See [#5414](https://github.com/DataDog/integrations-core/pull/5414).
+* [Added] Use lazy logging format. See [#5398](https://github.com/DataDog/integrations-core/pull/5398).
+* [Added] Add log filesystem container metric. See [#5383](https://github.com/DataDog/integrations-core/pull/5383).
+* [Added] Use lazy logging format. See [#5377](https://github.com/DataDog/integrations-core/pull/5377).
+* [Added] Add kubelet and runtime cpu and mem metrics. See [#5370](https://github.com/DataDog/integrations-core/pull/5370).
+* [Changed] Update metrics for >= 1.14. See [#5336](https://github.com/DataDog/integrations-core/pull/5336).
```

Comment:
Note: as mentioned by @AlexandreYang this PR seems to introduce support for 1.14+ metrics without breaking backwards compat, shis should probably be a `changelog/Added`.

Reply:
One metric is not present post-1.14, but that's only related to a change in k8s (changed from summary to histogram). => Rolling with a minor instead of a major.

Reply:
Reverted 4.0.0 release and pushed 3.5.0 with a release metadata update.